src/init.c

```diff
@@ -106,7 +106,7 @@ shmem_internal_shutdown_atexit(void)
     if ( shmem_internal_initialized && !shmem_internal_finalized &&
          !shmem_internal_initialized_with_start_pes && !shmem_internal_global_exit_called &&
          shmem_internal_my_pe == 0) {
-        fprintf(stderr, "Warning: shutting down without a call to shmem_finalize()\n");
+        RAISE_WARN_STR("shutting down without a call to shmem_finalize");
```

Comment:
Calling ```shmem_finalize``` is not compulsory in OpenSHMEM 1.3, so this is the right way to handle it for now.  We'll need to revisit this when we implement 1.4; we may want to change this to ```RETURN_ERROR_STR```.